sorts/bead_sort.py

```diff
@@ -0,0 +1,29 @@
+""" https://en.wikipedia.org/wiki/Bead_sort """
+
+
+def bead_sort(sequence: list) -> list:
+    """
+    >>> bead_sort([6,11, 12, 4, 1, 5])
```

Comment:
```suggestion
    >>> bead_sort([6, 11, 12, 4, 1, 5])
```

Reply:
Please test [1, .9, 0.0, 0, -1, -.9] and "Hello World!"

Reply:
I am wondering on "Hello world" ordering. Theory says that algorithm should be use for positive integers. This implementation works with `[1, .9, 0.0, 0, -1, -.9]` but should it support characters too?

Reply:
After a few moments, I decide to write two checks for it. I don't want to argue with the theory! :smile:  @cclauss maybe you have a better idea for this?